apps/dapp/src/components/Pages/Core/NewUI/TLC/TLCModal.tsx

```diff
@@ -69,7 +69,7 @@ export const TLCModal: React.FC<IProps> = ({ isOpen, onClose }) => {
   const [prices, setPrices] = useState<Prices>({ templePrice: 0, daiPrice: 0, tpi: 0 });
 
   const getPrices = async () => {
-    const { data } = await fetchGenericSubgraph(
```

Comment:
now we could add type check here, e.g. check `TxnHistoryTable.tsx` subgraph queries